.woodpecker.yml

```diff
@@ -16,35 +16,37 @@ pipeline:
     commands:
       - make test
 
-  build:
-    image: golang:1.16
-    commands:
-      - make build
-
   dryrun:
-    image: plugins/docker
-    dockerfile: docker/Dockerfile.linux.amd64
+    image: thegeeklab/drone-docker-buildx:latest
```

Comment:
@6543 Can / should we trust this image?

Reply:
Once that is available I can switch to it.  I've tried forking the drone-docker-buildx on my own and converting it to support woodpecker.  If there is already an image to start with I could assist to update that.

 https://codeberg.org/woodpecker-plugins/plugin-docker-buildx seems to be private.

Reply:
made it public